Sources/Commands/UserToolchain.swift

```diff
@@ -163,6 +148,34 @@ public struct UserToolchain: Toolchain {
         guard localFileSystem.isExecutableFile(clangCompiler) else {
             throw Error.invalidToolchain(problem: "could not find `clang` at expected path \(clangCompiler.asString)")
         }
+        return clangCompiler
+    }
+
+    public init(destination: Destination) throws {
+        self.destination = destination
+
+        // Get the search paths from PATH.
+        let envSearchPaths = getEnvSearchPaths(
```

Comment:
I think we should convert this into a property so we don't need to construct it again in the getClangCompiler method.

Reply:
done